docs/upgrading-stack.asciidoc

```diff
@@ -68,10 +68,6 @@ restart all nodes to maintain communication across the cluster.
 password (`changeme`). For more information, see
 {stack-ov}/built-in-users.html[Built-in users].
 
-. Stop any {ml} jobs that are running before starting the
-upgrade process. See
-{stack-ov}/stopping-ml.html[Stopping {ml}].
```

Comment:
Maybe instead of completely removing this section it could say something like "Consider stopping any {ml} jobs that are running - see [<close_ml.asciidoc>](https://github.com/elastic/elasticsearch/pull/38876/files#diff-131043b47d93bd06a9c3a94c8a6d7624) for the pros and cons of doing this".

Or do you think that's just too complicated for this guide?

Reply:
Thanks, I've altered that text.